docs/installation/additional-settings-for-developers/index.md

```diff
@@ -22,16 +22,46 @@ export RCUTILS_CONSOLE_OUTPUT_FORMAT="[{severity} {time}] [{name}]: {message} ({
 
 For more options, see [here](https://docs.ros.org/en/rolling/Tutorials/Logging-and-logger-configuration.html#console-output-formatting).
 
-### Enabling localhost-only communication
+## Network settings for ROS 2
+
+This section describes the network settings.
+ROS 2 employs DDS, and the configuration of ROS2 and DDS will be described separately.
+Please refer to [the official documentation](http://design.ros2.org/articles/ros_on_dds.html) for ROS2 networking concepts.
+
+### ROS 2 network setting
+
+ROS2 multicasts data on the local network by default. When developing in a company, etc., data flows over the local network and there is a possibility of collision.
+
+- Localhost-only communication
+- Same domain only communication on the local network
+
+Unless you plan to use multiple host computers on the local network, localhost-only communication is recommended.
+
+#### Enabling localhost-only communication
 
 By default, ROS 2 communicates using multi-cast, which may unnecessarily increase the network traffic.
 To avoid it, write the following in your `.bashrc`:
+For more information, see [here](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Configuring-ROS2-Environment.html#the-ros-localhost-only-variable).
 
 ```bash
 export ROS_LOCALHOST_ONLY=1
 ```
 
-### Setting up `ROS_DOMAIN_ID`
+If exported `ROS_LOCALHOST_ONLY=1`, `MULTICAST` must be enabled at the loopback address. Use the following command to verify that `MULTICAST` is included.
+
+```bash
+ip link show lo
+# Result sample
+# 1: lo: <LOOPBACK,MULTICAST,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+```
+
+If `MULTICAST` is not included, use the following command to enable it.
```

Comment:
To clarify it a bit more, how about this?

```suggestion
If the word `MULTICAST` is not printed, use the following command to enable it.
```

Reply:
Fixed at https://github.com/autowarefoundation/autoware-documentation/pull/306/commits/cebfd3c8f86dd3b92ebc9f1607a148f972c35a2b :+1: